src/signals/incident-management/containers/Dashboard/components/Filter/SelectList.tsx

```diff
@@ -129,14 +133,18 @@ const SelectList = ({ filterActiveName, setFilterActiveName }: Props) => {
         <RefreshIcon width={16} height={18} />
         Wis filters
       </Select>
-      <OptionListDropdown active={!!activeFilter?.name}>
-        {activeFilter?.name && isNumber(optionsOffsetLeftRef.current) && (
-          <OptionsList
-            optionsOffsetLeft={optionsOffsetLeftRef.current}
-            activeFilter={activeFilter}
-            setFilterNameActive={setFilterActiveName}
-          />
-        )}
+      <OptionListDropdown
+        active={!!activeFilter?.name}
```

Comment:
Double exclamation remarks? Why not just no remarks then (a quick google search didn't say anything about double exclamation remarks)?

Reply:
You should really fix your google. You're missing out!

<img width="771" alt="Screenshot 2023-02-07 at 12 01 39" src="https://user-images.githubusercontent.com/46756331/217227252-e80d8a61-75a0-425b-a2d8-465b9c0832dc.png">

Reply:
🥷

Reply:
I would if I knew how.